app/models/redirect_process.rb

```diff
@@ -0,0 +1,22 @@
+class RedirectProcess
```

Comment:
Renamed `RedirectService` to `RedirectProcess` to better describe the role this object fulfills. It essentially maps a token to a URL and does some accounting in the middle.